R/R/bridgestan.R

```diff
@@ -249,3 +263,13 @@ StanModel <- R6::R6Class("StanModel",
   ),
   cloneable=FALSE
 )
+#' Get and free the error message stored at the C++ pointer
+#' @keywords internal
+handle_error <- function(function_name, err_msg, err_ptr, lib_name) {
```

Comment:
Is it worth arranging the arguments in the same(-ish) order as in Julia and Python?  The other languages have `function_name` last and something-like `lib_name` first.